spec.bs

```diff
@@ -4732,6 +4823,11 @@ frequency capping.
   :: A [=string=]. A JSON serialized object corresponding to the ad that won the auction.
 </dl>
 
+A <dfn>seller capability</dfn> is a permission granted by an [=interest group=] to sellers. It's
+"`interest-group-counts`" or "`latency-stats`". Each value reprensents a single bit in a 4 bytes

```

Comment:
@caraitto how about this?
@domfarolino Our implementation uses enumSet to store this field. Does this description here make sense? We need the size when calculating the IG's size.